rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
```

Comment:
Because results are collected by parsing JSON [printed by benchmark script](https://github.com/ruby-bench/ruby-bench-suite/pull/8/files#diff-62a3b108fdf213cd836e342593ea887bR157), we want to have JSON-only output.
By default, IPS output is like this:
```
Calculating -------------------------------------
            addition    71.254k i/100ms
           addition2    68.658k i/100ms
           addition3    83.079k i/100ms
addition-test-long-label
                        70.129k i/100ms
-------------------------------------------------
            addition     4.955M (± 8.7%) i/s -     24.155M
           addition2    24.011M (± 9.5%) i/s -    114.246M
           addition3    23.958M (±10.1%) i/s -    115.064M
addition-test-long-label
                         5.014M (± 9.1%) i/s -     24.545M

```